lib/modules/platform/azure/index.ts

```diff
@@ -908,11 +911,20 @@ async function getUserIds(users: string[]): Promise<User[]> {
         if (ids.filter((c) => c.id === t.id).length === 0) {
           // TODO #22198
           ids.push({ id: t.id!, name: reviewer, isRequired });
+
+          validReviewers.push(reviewer);
         }
       }
     });
   });
 
+  users.forEach((u) => {
+    let reviewer = u.replace(requiredReviewerPrefix, '');
+    if (validReviewers.indexOf(reviewer) == -1) {
+      logger.warn(`${reviewer} is neither an Azure DevOps Team nor a user associated with a Team`);
```

Comment:
```suggestion
      logger.once.info(`${reviewer} is neither an Azure DevOps Team nor a user associated with a Team`);
```
I think warn is too hard here. 

@rarkins WDYT?